Packs/Okta/Integrations/Okta_IAM/README.md

```diff
@@ -19,8 +19,15 @@ For more information, please refer to the [Identity Lifecycle Management article
 | create-user-enabled | Create User Command Enabled | False |
 | update-user-enabled | Update User Command Enabled | False |
 | enable-disable-user-enabled | Enable/Disable User Commands Enabled | False |
+| create-if-not-exists | Automatically create user if not found in update and enable commands | False |
 | mapper-in | Incoming Mapper | True |
 | mapper-out | Outgoing Mapper | True |
+| max_fetch | Fetch Limit \(recommended less than 200\) | False |
+| isFetch | Fetch incidents | False |
+| incidentFetchInterval | Incidents Fetch Interval | False |
+| incidentType | Incident type | False |
+| fetch_query_filter | Fetch Query Filter | True |
```

Comment:
Provide more documentation, maybe link to the query syntax page. Where the user should query for these logs.
Maybe example